### ISSUE TITLE
Feature Request: Option to disable typewriter display of tutorial text.

### ISSUE THREAD

**Issue by rljacobson** [vasyop/miniC-hosting#1]
Awesome project! 

Would you consider an option to disable the typewriter-style display of the tutorial text? It might just be me, but I find it really distracting. 

Unrelated: The compile button obscures the code. I can open a new issue for it if you'd like. A simple fix would be to move the button to the top of the code column as if it were a header.
<img width="202" alt="screen shot 2019-02-09 at 1 34 33 pm" src="https://user-images.githubusercontent.com/1412598/52524831-7e17ed00-2c6f-11e9-8315-0dea86d5c862.png">

**Comment by vasyop** [comment]
I'm not sure what you have in mind. Do you mean just throwing the full wall of text at once?
I will see what I can do about the compile button. What resolution is that by the way?

**Comment by rljacobson** [comment]
I figured it would be easier to show you:
https://www.youtube.com/watch?v=KzP_gOkE_8k

**Comment by vasyop** [comment]
The compile button should adjust to the screen size now.

**Comment by rljacobson** [comment]
Better, but this is what I get. I was thinking we could find more horizontal space somewhere else, so I did some window resizing and found some other unexpected behavior that is likely not intended. I've included screenshots of that below as well. The last screenshot suggests that just reducing the height of the compile button might be a reasonable solution.
<img width="1298" alt="screen shot 2019-02-24 at 2 10 44 pm" src="https://user-images.githubusercontent.com/1412598/53303916-097fa980-383e-11e9-83cf-3e6a27bb6f50.png">
<img width="1314" alt="screen shot 2019-02-24 at 2 14 24 pm" src="https://user-images.githubusercontent.com/1412598/53304022-edc8d300-383e-11e9-834d-61370651376b.png">
<img width="1314" alt="screen shot 2019-02-24 at 2 15 39 pm" src="https://user-images.githubusercontent.com/1412598/53304031-f3261d80-383e-11e9-9373-cff000023e96.png">
